docs/checkit.js

```diff
@@ -91,6 +94,58 @@ function CheckIt() {
 
 };
 
+CheckIt.prototype.addMonth = function() {
+    // Add a single month to the calendar
+    // TODO
+    //create the month object
+    // Need a dateString, possible end Date from cal
+    // Maybe grab the last month in the page, and that's how you can get
+    // the month and year for the new month, end it on whatever endDate
+    // user picked for the previous month (then previous month would need to be
+    // filled all the way too. 
+    //newMonth = new Month(
+    
+    
+    //this.attachCellClickHandler([newMonth]);
+};
+
+CheckIt.prototype.attachCellClickHandler = function(monthObjArray) {
+    // Takes an array of month objects (the array can be of length 1)
+    var checkitObj = this;
+    
+    monthObjArray.forEach( function(monthObj) {
+        var $monthDiv = $(`#${monthObj.monthId}`);
```

Comment:
prefer normal strings over backticks unless you're getting a lot of bang for your buck.

```
`${user}-${host}-${ip}.data.log`
```
for example